seshdash/views.py

```diff
@@ -575,6 +574,28 @@ def get_alerts(request):
 
     return HttpResponse(json.dumps(alert_data))
 
+    
+
+
+def get_notifications_alerts(request):
+    
+    sites = Sesh_Site.objects.all()
```

Comment:
We can't use `Sesh_Site.objects.all()` you need to fetch the sites that the user has permission to see.